src/lucky_record/query_builder.cr

```diff
@@ -75,6 +75,13 @@ class LuckyRecord::QueryBuilder
     self
   end
 
+  def select(selection : Array(Symbol))
```

Comment:
I like this type restriction 👍 That way we can later add a `select(selection : String)` that can be used to select just about anything